src/test/ovmsconfig_test.cpp

```diff
@@ -1997,6 +1998,30 @@ TEST_F(OvmsParamsTest, hostname_ip_regex) {
     EXPECT_EQ(ovms::Config::check_hostname_or_ip("::ffff:192.0.2.128"), true);
     //  IPv4-translated IPv6 addresses
     EXPECT_EQ(ovms::Config::check_hostname_or_ip("::ffff:0:192.0.2.128"), true);
+
+    // Multiple selections
+    EXPECT_EQ(ovms::Config::check_hostname_or_ip("0.0.0.0"), true);
```

Comment:
127.0.0.1,::1 as actual e2e unit test?

Reply:
not all build environments support ipv6, therefore this is bad idea to make such e2e test